src/main/java/org/isf/lab/rest/LaboratoryController.java

```diff
@@ -646,5 +650,31 @@ public ResponseEntity<LabWithRowsDTO> getExamWithRowsById(@PathVariable Integer
 		lab.setLaboratoryRowList(labDescription);
 		return ResponseEntity.ok(lab);
 	}
-
+	
+	/**
+	 * Set an {@link Laboratory} record to deleted.
+	 * 
+	 * @param code
+	 * @return {@code true} if the record has been set to invalid, {@code false} otherwise.
+	 * @throws OHServiceException
+	 * @author Arnaud
+	 */
+	@DeleteMapping(value = "/laboratories/examRequest/{code}", produces = MediaType.APPLICATION_JSON_VALUE)
+	public ResponseEntity<Boolean> deleteExamRequest(@PathVariable Integer code) throws OHServiceException {
+		LOGGER.info("Get Laboratory associated to specified CODE: {}", code);
+		Optional<Laboratory> labo = laboratoryManager.getLaboratory(code);
+		Laboratory lab = new Laboratory();
+		if (labo.isPresent()) {
+			lab = labo.get();
+			if (!lab.getStatus().toUpperCase().equalsIgnoreCase(draft) && !lab.getStatus().toUpperCase().equalsIgnoreCase(open)) {
```

Comment:
why do `toUpperCase()` and then use the comparison `equalsIgnoreCase()`?     Can't you just drop the `toUppserCase()`?